### PR TITLE
Update assigned tasks docs

### PR DESCRIPTION
**Pull request by emintham**
- project and application_id are now returned.
- project is expandable.

Refs #SDE-2555

### CODE REVIEW

File: source/includes/assigned_tasks.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ Content-Type: application/json
         "title": "Secure forgotten password",
         "updated": "2015-07-08T02:16:33.923315Z",
         "url": "http://example.com/bunits/bu1/app1/proj1/tasks/phase/requirements/1-T2",
-        "verification_status": null
+        "verification_status": null,
+        "project": 1,
+        "application_id": 1
```
Comment (RouganStriker) [inline source/includes/assigned_tasks.md]: There are some inconsistencies in our docs with the ```_id``` suffix; sometimes we include it sometimes we don't but it always refers to the application id

**Comment by emintham** [comment]
Seems the case without _id is more common. The only other place with _id is in the applications docs to GET a specific application in the URL. I've changed this so that it does not have _id.

**Comment by RouganStriker** [comment]
@emintham are you going to rename it on sde as well? I think the API currently uses application_id

**Comment by emintham** [comment]
Yup it's been renamed on SDE side. Running tests right now before I merge to make sure I don't break anything. Is the widget using the application_id at the moment?

**Pull request merged**